Description: Disable building of some internal tools that no longer link
	because internal symbols are no longer exported.

diff --git a/Makefile.am b/Makefile.am
index b4098f8..0d66934 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -8,13 +8,6 @@ if ENABLE_DEC265
 SUBDIRS+=dec265
 endif
 
-if ENABLE_ENCODER
-SUBDIRS+=enc265
-endif
-
-SUBDIRS+=tools
-SUBDIRS+=acceleration-speed
-
 if ENABLE_SHERLOCK265
 SUBDIRS+=sherlock265
 endif
